### PR TITLE
fix doc builds after recent changes

### PR DESCRIPTION
**Pull request by pmeier**
This is to get the CI green again on the `corpus-dev` branch. We still need to add actual documentation for the changes we have made. I'll flag some of them below.

### CODE REVIEW

File: ragna/core/_rag.py [hunk 1]
```diff
@@ -241,11 +268,22 @@ async def answer(self, prompt: str, *, stream: bool = False) -> Message:
...
+            str,
+            Path,
+            Collection[Union[Document, str, Path]],
+        ],
     ) -> tuple[Optional[list[Document]], Optional[MetadataFilter], bool]:
         if isinstance(input, MetadataFilter) or input is None:
             return None, input, True
 
+        if not isinstance(input, collections.abc.Collection):
+            input = [input]
```
Comment (pmeier) [inline ragna/core/_rag.py:285]: Driveby to also allow single documents.

File: ragna/core/_rag.py [hunk 2]
```diff
@@ -82,21 +83,34 @@ def _load_component(
...
+            Document,
+            str,
+            Path,
+            Collection[Union[Document, str, Path]],
+        ],
         *,
         source_storage: Union[Type[SourceStorage], SourceStorage],
         assistant: Union[Type[Assistant], Assistant],
-        corpus_name: Optional[str],
+        corpus_name: Optional[str] = None,
```
Comment (pmeier) [inline ragna/core/_rag.py:97]: @nenb I've added a default value here, because I don't want to force users to always set this one.

File: docs/examples/gallery_streaming.py [hunk 3]
```diff
@@ -119,9 +119,10 @@ def answer(self, messages):
         "/chats",
         json={
             "name": "Tutorial REST API",
-            "documents": [document],
+            "input": [document],
             "source_storage": source_storages.RagnaDemoSourceStorage.display_name(),
             "assistant": DemoStreamingAssistant.display_name(),
+            "corpus_name": None,
```
Comment (pmeier) [inline docs/examples/gallery_streaming.py:125]: We probably want to have this as default value as well.

Comment (nenb) [reply docs/examples/gallery_streaming.py:125]: Is this still valid after your comments in #493?

**Review by nenb — APPROVED** [review]
Seems fine, but I'm not 💯 sure if you want to set the `corpus_name` as `None` or use a sentinel after #493.

**Comment by pmeier** [comment]
Superseded by #498.
